exercises/yacht/stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
+# You should not edit this file by hand.
+# For more information, please see the documentation at:
+#   https://docs.haskellstack.org/en/stable/lock_files
+
+packages: []
+snapshots:
+- completed:
+    size: 523443
+    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/14/0.yaml
+    sha256: 283773e7120f5446d961eab35ea95c9af9c24187cc178537bd29273200a05171
+  original: lts-14.0
```

Comment:
This file can be removed; it isn't present in any of the other exercises, either.

Reply:
Oh, I didn't realize this wasn't in the repo-wide `.gitignore`.

Consider it gone.